configs/templates/gsi-addon-dev/spack.yaml

```diff
@@ -0,0 +1,25 @@
+spack:
+  concretizer:
+    unify: when_possible
+
+  view: false
+  include:
+  - site
+  - common
+  definitions:
+  # Note: Set 'compilers' manually; must match upstream list
+  - compilers: []
+  - packages:
+    - global-workflow-env ^bufr@11.7.0 ^metplus@3.1.1 ^met@9.1.3
+    - ufs-weather-model-env
+    - gsi-env
+
+  specs:
+  - matrix:
+    - [$packages]
+    - [$compilers]
+
+  # Note: Set 'install_tree' to the upstream unified-env install path
+  upstreams:
+    spack-stack-unified-env:
+       install_tree: /path/to/envs/unified-env/install
```

Comment:
This should probably come from the `spack-stack create env --upstream` command. I will test this PR when I create the release candidate for spack-stack-1.6.0 on Orion or Hercules (most likely the latter).

Reply:
I confirmed that by using `spack stack create env --template=gsi-addon --upstream=/whatever/your/path/to/envs/unified-env/install --site=pick-your-site` creates lines 22-25, so please remove those

Reply:
And, if you like, leave a comment there that explains that upstreams will be added with the `--upstream` option for `spack stack create env`? But not absolutely necessary, as you wish.

Reply:
Fixed.  Thanks for checking!